app/mentors.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useContext, useEffect, useState } from "react";
+import { View, StyleSheet, Linking, ActivityIndicator } from "react-native";
+import { GestureHandlerRootView, ScrollView } from "react-native-gesture-handler";
+import { Mentor } from "./types";
+import { db } from "@/firebaseConfig";
+import { collection, getDocs } from "firebase/firestore";
+import { Card, Button, Text, Avatar } from "@rneui/themed";
+import { AuthContext } from "@/app/index";
+import Login from "@/app/login";
+
+// each mentor has:
+// 	    name : string;
+// 	    pictureURL : string;
+// 	    interests: string[];
+// 	    company : string;
+// 	    associationDate : Date;
+// 	    email : string;
+
+export default function Mentors() {
```

Comment:
Schimba aici in arrow function si pune export default la final ca la celelalte pagini ca sa pastram consistenta.